### ISSUE TITLE
Update default docker settings

### ISSUE THREAD

**Issue by humitos** [rtfd/readthedocs.org#5153]
We need to apply some changes to the default values of our Docker settings to start using latest released images by default.

Reference: https://github.com/rtfd/readthedocs.org/pull/5056#pullrequestreview-193700263
Depends on #5056 being merged

**Comment by stsewd** [comment]
This was already done https://github.com/rtfd/readthedocs.org/pull/5155
